README.md

```diff
@@ -10,12 +10,16 @@ Under construction!
 1. Make a virtual environment and source into it
 1. `pip install -r requirements.dev.txt`
 1. `npm install -g npm-watch`
+1. `npm install -g riot`
```

Comment:
can we do this all in one line? `npm install -g npm-watch riot stylus`?

Reply:
Updated this to all be one line